docs/source/concepts/model_specification/index.rst

```diff
@@ -24,6 +24,11 @@ You can find a short intro to yaml basics
    :local:
    :backlinks: none
 
+The Plugins Block
+-----------------
+
+TBD
```

Comment:
This seems lower priority than other TBDs floating around our docs so I figured leaving it this way is fine. But I _did_ want to reference the missing configuration block docs so I added those below